app/views/calls/create.js.erb

```diff
@@ -1 +1,16 @@
-$(".new-call").modal('hide');
+
+
+// Fires on call complete POST
+
+// TODO: write if statement for url ending in /edit/
+    $("#call_log").html("<%= escape_javascript(render partial: 'patients/call_log') %>")
+      .promise().done(function() {
+        $(".new-call").modal('hide')
+        $('body').removeClass('modal-open')
+        $('.modal-backdrop').remove()
+      });
+if (location.pathname == "/") {
```

Comment:
In the spirit of rails, can we turn this into a `path` variable? Dropping in something like `<%= authenticated_root_path %>` instead of `/` as the url will help keep this code nimble.